go.sum

```diff
@@ -12,6 +12,10 @@ github.com/coreos/go-iptables v0.3.0 h1:UTQkjHl9rPwwtXZhXbY3T932cV9aUnKlSsZ7YGfJ
 github.com/coreos/go-iptables v0.3.0/go.mod h1:/mVI274lEDI2ns62jHCDnCyBF9Iwsmekav8Dbxlm1MU=
 github.com/coreos/go-iptables v0.4.1 h1:TyEMaK2xD/EcB0385QcvX/OvI2XI7s4SJEI2EhZFfEU=
 github.com/coreos/go-iptables v0.4.1/go.mod h1:/mVI274lEDI2ns62jHCDnCyBF9Iwsmekav8Dbxlm1MU=
+github.com/coreos/go-iptables v0.4.2-0.20190627140312-f1d0510cabcb h1:9LWrtjSwtaPzlx8rZ5c5OGhJVMyJh7YhfFYypzwCAfM=
```

Comment:
I think we need a **"go mod tidy"** to remove these useless lines.

Reply:
agreed - done